### ISSUE TITLE
shutdown not working

### ISSUE THREAD

**Issue by jensopetersen** [eXist-db/dashboard#37]
At 9eaab9, when shutting down through the dashboard, Jetty stops, but eXist is still running:

`ps ax | grep 'exist'
61371 s000  S+     0:00.00 /bin/bash /Applications/exist_dev/bin/startup.sh
61380 s000  S+     0:21.96 /Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/bin/java -Xms128m -Xmx1024m -Dfile.encoding=UTF-8 -Djava.endorsed.dirs=/Applications/exist_dev/lib/endorsed -Dexist.home=/Applications/exist_dev -jar /Applications/exist_dev/start.jar jetty
61429 s001  S+     0:00.00 grep exist`

**Comment by adamretter** [comment]
Okay I am going to need some help reproducing this, as what I tried here seems to work very well. So my steps were:

1. Start eXist

```
hollowcore:exist-git aretter$ bin/startup.sh 

04 Jun 2015 12:18:26,179 [main] INFO  (JettyStart.java [run]:121) - Configuring eXist from /Users/aretter/NetBeansProjects/exist-git/conf.xml 
04 Jun 2015 12:18:26,180 [main] INFO  (JettyStart.java [run]:123) - Running with Java 1.8.0_45 [Oracle Corporation (Java HotSpot(TM) 64-Bit Server VM) in /Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre] 
04 Jun 2015 12:18:26,180 [main] INFO  (JettyStart.java [run]:129) - Running as user 'aretter' 
04 Jun 2015 12:18:26,183 [main] INFO  (JettyStart.java [run]:132) - [eXist Version : 2.2] 
04 Jun 2015 12:18:26,184 [main] INFO  (JettyStart.java [run]:134) - [eXist Build : 20150604] 
04 Jun 2015 12:18:26,184 [main] INFO  (JettyStart.java [run]:136) - [eXist Home : unknown] 
04 Jun 2015 12:18:26,184 [main] INFO  (JettyStart.java [run]:138) - [Git commmit : develop-9eaab9b] 
04 Jun 2015 12:18:26,185 [main] INFO  (JettyStart.java [run]:141) - [Operating System : Mac OS X 10.10.3 x86_64] 
04 Jun 2015 12:18:26,185 [main] INFO  (JettyStart.java [run]:146) - [jetty.home : /Users/aretter/NetBeansProjects/exist-git/tools/jetty] 
04 Jun 2015 12:18:26,185 [main] INFO  (JettyStart.java [run]:148) - [log4j.configurationFile : file:/Users/aretter/NetBeansProjects/exist-git/log4j2.xml] 
04 Jun 2015 12:18:38,068 [main] INFO  (JettyStart.java [lifeCycleStarting]:414) - Jetty server starting... 
04 Jun 2015 12:18:41,779 [main] INFO  (JettyStart.java [lifeCycleStarted]:422) - Jetty server started. 
04 Jun 2015 12:18:41,800 [main] INFO  (JettyStart.java [run]:234) - ----------------------------------------------------- 
04 Jun 2015 12:18:41,801 [main] INFO  (JettyStart.java [run]:235) - Server has started on ports 8080 8443. Configured contexts: 
04 Jun 2015 12:18:41,801 [main] INFO  (JettyStart.java [run]:244) - '/exist' 
04 Jun 2015 12:18:41,806 [main] INFO  (JettyStart.java [run]:260) - '/exist/openid' 
04 Jun 2015 12:18:41,811 [main] INFO  (JettyStart.java [run]:276) - '/exist/oauth' 
04 Jun 2015 12:18:41,812 [main] INFO  (JettyStart.java [run]:282) - -----------------------------------------------------
```

2. Go to the dashboard at `http://localhost:8080/exist/apps/dashboard/index.html` and click the big shutdown button, log in as the `admin` user and accept the shutdown. The following logs are written:

```
04 Jun 2015 12:19:24,115 [Timer-0] INFO  (JettyStart.java [shutdown]:377) - Database shutdown: stopping server in 1sec ... 
04 Jun 2015 12:19:25,120 [jetty shutdown schedule] INFO  (JettyStart.java [lifeCycleStopping]:433) - Jetty server stopping... 
04 Jun 2015 12:19:26,257 [jetty shutdown schedule] INFO  (JettyStart.java [lifeCycleStopped]:439) - Jetty server stopped
```

3. Check whether any `exist` process is still running:
```
hollowcore:exist-git aretter$ ps -ax | grep exist
75483 ttys004    0:00.01 grep exist
```

So what am I missing? Can anyone confirm/refute @jensopetersen issue?

**Comment by zwobit** [comment]
I can confirm the issue on exist-db/develop from today with linux.



```
08 Jun 2015 13:43:06,840 [jetty shutdown schedule] INFO  (JettyStart.java [lifeCycleStopping]:433) - Jetty server stopping... 
08 Jun 2015 13:43:07,953 [jetty shutdown schedule] INFO  (JettyStart.java [lifeCycleStopped]:439) - Jetty server stopped 
```

```
zwobit    4152 60.2  1.2 5651660 415200 pts/2  Sl+  13:42   0:27 /home/zwobit/.gentoo/java-config-2/current-user-vm/bin/java -Xms128m -Xmx1024m -Dfile.encoding=UTF-8 -Djava.endorsed.dirs=/home/zwobit/work/oss/zwobit-exist/bin/../lib/endorsed -Dexist.home=/home/zwobit/work/oss/zwobit-exist/bin/.. -jar /home/zwobit/work/oss/zwobit-exist/bin/../start.jar jetty
```

**Comment by adamretter** [comment]
@zwobit Any chance you can post a stack dump of the zombie process, e.g. `kill -QUIT psid` of `jstack psid`?

**Comment by zwobit** [comment]
Sure. kill -QUIT:

```
08 Jun 2015 14:04:28,572 [main] INFO  (JettyStart.java [run]:234) - ----------------------------------------------------- 
08 Jun 2015 14:04:28,572 [main] INFO  (JettyStart.java [run]:235) - Server has started on ports 8080 8443. Configured contexts: 
08 Jun 2015 14:04:28,572 [main] INFO  (JettyStart.java [run]:244) - '/exist' 
08 Jun 2015 14:04:28,572 [main] INFO  (JettyStart.java [run]:282) - ----------------------------------------------------- 
2015-06-08 14:04:44
Full thread dump Java HotSpot(TM) 64-Bit Server VM (25.45-b02 mixed mode):

"DestroyJavaVM" #39 prio=5 os_prio=0 tid=0x00007f101c00a000 nid=0x2af8 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Scanner-1" #37 daemon prio=5 os_prio=0 tid=0x00007f101d999800 nid=0x2b36 in Object.wait() [0x00007f0ff50b8000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000ecbfbf58> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"HashSessionScavenger-2" #36 daemon prio=5 os_prio=0 tid=0x00007f101d998800 nid=0x2b34 in Object.wait() [0x00007f0ff51b9000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000ecbe4148> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"HashSessionScavenger-1" #35 daemon prio=5 os_prio=0 tid=0x00007f101d996000 nid=0x2b33 in Object.wait() [0x00007f0ff52ba000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000ebe600d0> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"Scanner-0" #34 daemon prio=5 os_prio=0 tid=0x00007f101d990000 nid=0x2b31 in Object.wait() [0x00007f0ff53bb000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000eaef3300> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"eXistThread-33" #33 prio=5 os_prio=0 tid=0x00007f101d98d800 nid=0x2b30 waiting on condition [0x00007f0ff54bc000]
   java.lang.Thread.State: TIMED_WAITING (parking)
        at sun.misc.Unsafe.park(Native Method)
        - parking to wait for  <0x00000000eae5ed20> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
        at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
        at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2078)
        at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:342)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.idleJobPoll(QueuedThreadPool.java:526)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.access$600(QueuedThreadPool.java:44)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:572)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-32" #32 prio=5 os_prio=0 tid=0x00007f101d98b800 nid=0x2b2f waiting on condition [0x00007f0ff55bd000]
   java.lang.Thread.State: TIMED_WAITING (parking)
        at sun.misc.Unsafe.park(Native Method)
        - parking to wait for  <0x00000000eae5ed20> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
        at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
        at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2078)
        at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:342)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.idleJobPoll(QueuedThreadPool.java:526)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.access$600(QueuedThreadPool.java:44)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:572)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-31 Acceptor1 SslSelectChannelConnector@0.0.0.0:8443" #31 prio=5 os_prio=0 tid=0x00007f101d989800 nid=0x2b2e waiting for monitor entry [0x00007f0ff56be000]
   java.lang.Thread.State: BLOCKED (on object monitor)
        at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:234)
        - waiting to lock <0x00000000ed795650> (a java.lang.Object)
        at org.eclipse.jetty.server.nio.SelectChannelConnector.accept(SelectChannelConnector.java:109)
        at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:938)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-30 Acceptor0 SslSelectChannelConnector@0.0.0.0:8443" #30 prio=5 os_prio=0 tid=0x00007f101d987800 nid=0x2b2d runnable [0x00007f0ff57bf000]
   java.lang.Thread.State: RUNNABLE
        at sun.nio.ch.ServerSocketChannelImpl.accept0(Native Method)
        at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:422)
        at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)
        - locked <0x00000000ed795650> (a java.lang.Object)
        at org.eclipse.jetty.server.nio.SelectChannelConnector.accept(SelectChannelConnector.java:109)
        at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:938)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-29 Selector0" #29 prio=5 os_prio=0 tid=0x00007f101d985800 nid=0x2b2c runnable [0x00007f0ff58c0000]
   java.lang.Thread.State: RUNNABLE
        at sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
        at sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
        at sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:79)
        at sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
        - locked <0x00000000ed795da0> (a sun.nio.ch.Util$2)
        - locked <0x00000000ed795d90> (a java.util.Collections$UnmodifiableSet)
        - locked <0x00000000ed795c78> (a sun.nio.ch.EPollSelectorImpl)
        at sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
        at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:569)
        at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-28 Acceptor1 SelectChannelConnector@0.0.0.0:8080" #28 prio=5 os_prio=0 tid=0x00007f101d984000 nid=0x2b2b runnable [0x00007f0ff59c1000]
   java.lang.Thread.State: RUNNABLE
        at sun.nio.ch.ServerSocketChannelImpl.accept0(Native Method)
        at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:422)
        at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)
        - locked <0x00000000ecbfe970> (a java.lang.Object)
        at org.eclipse.jetty.server.nio.SelectChannelConnector.accept(SelectChannelConnector.java:109)
        at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:938)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-27 Acceptor0 SelectChannelConnector@0.0.0.0:8080" #27 prio=5 os_prio=0 tid=0x00007f101d97f800 nid=0x2b2a waiting for monitor entry [0x00007f0ff5ac2000]
   java.lang.Thread.State: BLOCKED (on object monitor)
        at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:234)
        - waiting to lock <0x00000000ecbfe970> (a java.lang.Object)
        at org.eclipse.jetty.server.nio.SelectChannelConnector.accept(SelectChannelConnector.java:109)
        at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:938)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-26 Selector1" #26 prio=5 os_prio=0 tid=0x00007f101d97d800 nid=0x2b29 runnable [0x00007f0ff5bc3000]
   java.lang.Thread.State: RUNNABLE
        at sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
        at sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
        at sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:79)
        at sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
        - locked <0x00000000ecc7a048> (a sun.nio.ch.Util$2)
        - locked <0x00000000ecc7a038> (a java.util.Collections$UnmodifiableSet)
        - locked <0x00000000ecc79f20> (a sun.nio.ch.EPollSelectorImpl)
        at sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
        at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:569)
        at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-25 Selector1" #25 prio=5 os_prio=0 tid=0x00007f101d97b800 nid=0x2b28 runnable [0x00007f0ff5cc4000]
   java.lang.Thread.State: RUNNABLE
        at sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
        at sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
        at sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:79)
        at sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
        - locked <0x00000000ed7971e8> (a sun.nio.ch.Util$2)
        - locked <0x00000000ed7971d8> (a java.util.Collections$UnmodifiableSet)
        - locked <0x00000000ed7970c0> (a sun.nio.ch.EPollSelectorImpl)
        at sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
        at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:569)
        at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"eXistThread-24 Selector0" #24 prio=5 os_prio=0 tid=0x00007f101d97a000 nid=0x2b27 runnable [0x00007f0ff5dc5000]
   java.lang.Thread.State: RUNNABLE
        at sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
        at sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
        at sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:79)
        at sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
        - locked <0x00000000ecc785b0> (a sun.nio.ch.Util$2)
        - locked <0x00000000ecc78528> (a java.util.Collections$UnmodifiableSet)
        - locked <0x00000000ecc783a8> (a sun.nio.ch.EPollSelectorImpl)
        at sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
        at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:569)
        at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
        at java.lang.Thread.run(Thread.java:745)

"org.eclipse.jetty.util.RolloverFileOutputStream" #23 daemon prio=5 os_prio=0 tid=0x00007f101d945800 nid=0x2b26 in Object.wait() [0x00007f0ff5ec6000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000eae5a290> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"HashSessionScavenger-0" #22 daemon prio=5 os_prio=0 tid=0x00007f101d8bd000 nid=0x2b23 in Object.wait() [0x00007f0ff6ec8000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000f2e2dbc8> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"Thread-3" #19 prio=5 os_prio=0 tid=0x00007f101cf01000 nid=0x2b1e in Object.wait() [0x00007f0ff6bc7000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.Object.wait(Object.java:502)
        at org.exist.storage.journal.FileSyncThread.run(FileSyncThread.java:117)
        - locked <0x00000000c0b06598> (a org.exist.storage.journal.FileSyncThread)

"DefaultQuartzScheduler_QuartzSchedulerThread" #17 prio=5 os_prio=0 tid=0x00007f101ceb1000 nid=0x2b1c in Object.wait() [0x00007f0ff6fc9000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at org.quartz.core.QuartzSchedulerThread.run(QuartzSchedulerThread.java:311)
        - locked <0x00000000c0d70248> (a java.lang.Object)

"DefaultQuartzScheduler_Worker-4" #16 prio=5 os_prio=0 tid=0x00007f101ce95000 nid=0x2b1b in Object.wait() [0x00007f0ff70ca000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
        - locked <0x00000000c0c90068> (a java.lang.Object)

"DefaultQuartzScheduler_Worker-3" #15 prio=5 os_prio=0 tid=0x00007f101ce93000 nid=0x2b1a in Object.wait() [0x00007f0ff71cb000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
        - locked <0x00000000c0c8fd90> (a java.lang.Object)

"DefaultQuartzScheduler_Worker-2" #14 prio=5 os_prio=0 tid=0x00007f101ce91800 nid=0x2b19 in Object.wait() [0x00007f0ff72cc000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
        - locked <0x00000000c0c8fab8> (a java.lang.Object)

"DefaultQuartzScheduler_Worker-1" #13 prio=5 os_prio=0 tid=0x00007f101ce8b800 nid=0x2b16 in Object.wait() [0x00007f0ff73cd000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
        - locked <0x00000000c0c8f7e0> (a java.lang.Object)

"Service Thread" #9 daemon prio=9 os_prio=0 tid=0x00007f101c237000 nid=0x2b0b runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C1 CompilerThread3" #8 daemon prio=9 os_prio=0 tid=0x00007f101c229800 nid=0x2b0a waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread2" #7 daemon prio=9 os_prio=0 tid=0x00007f101c227800 nid=0x2b07 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread1" #6 daemon prio=9 os_prio=0 tid=0x00007f101c225800 nid=0x2b06 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread0" #5 daemon prio=9 os_prio=0 tid=0x00007f101c223000 nid=0x2b05 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Signal Dispatcher" #4 daemon prio=9 os_prio=0 tid=0x00007f101c221800 nid=0x2b04 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Finalizer" #3 daemon prio=8 os_prio=0 tid=0x00007f101c1e9000 nid=0x2b03 in Object.wait() [0x00007f10051f0000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
        - locked <0x00000000c08ad820> (a java.lang.ref.ReferenceQueue$Lock)
        at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
        at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:209)

"Reference Handler" #2 daemon prio=10 os_prio=0 tid=0x00007f101c1e7000 nid=0x2b02 in Object.wait() [0x00007f10052f1000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.Object.wait(Object.java:502)
        at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:157)
        - locked <0x00000000c0da3e48> (a java.lang.ref.Reference$Lock)

"VM Thread" os_prio=0 tid=0x00007f101c1e2000 nid=0x2b01 runnable 

"GC task thread#0 (ParallelGC)" os_prio=0 tid=0x00007f101c020000 nid=0x2af9 runnable 

"GC task thread#1 (ParallelGC)" os_prio=0 tid=0x00007f101c021800 nid=0x2afa runnable 

"GC task thread#2 (ParallelGC)" os_prio=0 tid=0x00007f101c023800 nid=0x2afb runnable 

"GC task thread#3 (ParallelGC)" os_prio=0 tid=0x00007f101c025000 nid=0x2afc runnable 

"GC task thread#4 (ParallelGC)" os_prio=0 tid=0x00007f101c027000 nid=0x2afd runnable 

"GC task thread#5 (ParallelGC)" os_prio=0 tid=0x00007f101c028800 nid=0x2afe runnable 

"GC task thread#6 (ParallelGC)" os_prio=0 tid=0x00007f101c02a800 nid=0x2aff runnable 

"GC task thread#7 (ParallelGC)" os_prio=0 tid=0x00007f101c02c000 nid=0x2b00 runnable 

"VM Periodic Task Thread" os_prio=0 tid=0x00007f101c239800 nid=0x2b0e waiting on condition 

JNI global references: 298

Heap
 PSYoungGen      total 136192K, used 62998K [0x00000000eab00000, 0x00000000f4300000, 0x0000000100000000)
  eden space 131072K, 44% used [0x00000000eab00000,0x00000000ee38b9f0,0x00000000f2b00000)
  from space 5120K, 99% used [0x00000000f2d00000,0x00000000f31fa1f8,0x00000000f3200000)
  to   space 12288K, 0% used [0x00000000f3700000,0x00000000f3700000,0x00000000f4300000)
 ParOldGen       total 87552K, used 16286K [0x00000000c0000000, 0x00000000c5580000, 0x00000000eab00000)
  object space 87552K, 18% used [0x00000000c0000000,0x00000000c0fe7b88,0x00000000c5580000)
 Metaspace       used 33317K, capacity 33718K, committed 33792K, reserved 1079296K
  class space    used 4196K, capacity 4336K, committed 4352K, reserved 1048576K

```

**Comment by zwobit** [comment]
@adamretter and after "shutdown":
08 Jun 2015 14:09:01,848 [Timer-0] INFO  (JettyStart.java [shutdown]:377) - Database shutdown: stopping server in 1sec ... 
08 Jun 2015 14:09:02,848 [jetty shutdown schedule] INFO  (JettyStart.java [lifeCycleStopping]:433) - Jetty server stopping... 
08 Jun 2015 14:09:03,954 [jetty shutdown schedule] INFO  (JettyStart.java [lifeCycleStopped]:439) - Jetty server stopped 
2015-06-08 14:09:06
Full thread dump Java HotSpot(TM) 64-Bit Server VM (25.45-b02 mixed mode):

"jetty shutdown schedule" #42 daemon prio=5 os_prio=0 tid=0x00007f0f5c002000 nid=0x2e6d in Object.wait() [0x00007f0ff6bc7000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.Object.wait(Object.java:502)
        at java.util.TimerThread.mainLoop(Timer.java:526)
        - locked <0x00000000ecfb8190> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"Timer-0" #40 prio=5 os_prio=0 tid=0x00007f0f68038800 nid=0x2e67 in Object.wait() [0x00007f0ff455a000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.Object.wait(Object.java:502)
        at java.util.TimerThread.mainLoop(Timer.java:526)
        - locked <0x00000000ece90518> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"DestroyJavaVM" #39 prio=5 os_prio=0 tid=0x00007f101c00a000 nid=0x2af8 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"org.eclipse.jetty.util.RolloverFileOutputStream" #23 daemon prio=5 os_prio=0 tid=0x00007f101d945800 nid=0x2b26 in Object.wait() [0x00007f0ff5ec6000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.util.TimerThread.mainLoop(Timer.java:552)
        - locked <0x00000000c0ceaaf0> (a java.util.TaskQueue)
        at java.util.TimerThread.run(Timer.java:505)

"Service Thread" #9 daemon prio=9 os_prio=0 tid=0x00007f101c237000 nid=0x2b0b runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C1 CompilerThread3" #8 daemon prio=9 os_prio=0 tid=0x00007f101c229800 nid=0x2b0a waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread2" #7 daemon prio=9 os_prio=0 tid=0x00007f101c227800 nid=0x2b07 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread1" #6 daemon prio=9 os_prio=0 tid=0x00007f101c225800 nid=0x2b06 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread0" #5 daemon prio=9 os_prio=0 tid=0x00007f101c223000 nid=0x2b05 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Signal Dispatcher" #4 daemon prio=9 os_prio=0 tid=0x00007f101c221800 nid=0x2b04 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Finalizer" #3 daemon prio=8 os_prio=0 tid=0x00007f101c1e9000 nid=0x2b03 in Object.wait() [0x00007f10051f0000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
        - locked <0x00000000c0547418> (a java.lang.ref.ReferenceQueue$Lock)
        at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
        at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:209)

"Reference Handler" #2 daemon prio=10 os_prio=0 tid=0x00007f101c1e7000 nid=0x2b02 in Object.wait() [0x00007f10052f1000]
   java.lang.Thread.State: WAITING (on object monitor)
        at java.lang.Object.wait(Native Method)
        at java.lang.Object.wait(Object.java:502)
        at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:157)
        - locked <0x00000000c0a08d10> (a java.lang.ref.Reference$Lock)

"VM Thread" os_prio=0 tid=0x00007f101c1e2000 nid=0x2b01 runnable 

"GC task thread#0 (ParallelGC)" os_prio=0 tid=0x00007f101c020000 nid=0x2af9 runnable 

"GC task thread#1 (ParallelGC)" os_prio=0 tid=0x00007f101c021800 nid=0x2afa runnable 

"GC task thread#2 (ParallelGC)" os_prio=0 tid=0x00007f101c023800 nid=0x2afb runnable 

"GC task thread#3 (ParallelGC)" os_prio=0 tid=0x00007f101c025000 nid=0x2afc runnable 

"GC task thread#4 (ParallelGC)" os_prio=0 tid=0x00007f101c027000 nid=0x2afd runnable 

"GC task thread#5 (ParallelGC)" os_prio=0 tid=0x00007f101c028800 nid=0x2afe runnable 

"GC task thread#6 (ParallelGC)" os_prio=0 tid=0x00007f101c02a800 nid=0x2aff runnable 

"GC task thread#7 (ParallelGC)" os_prio=0 tid=0x00007f101c02c000 nid=0x2b00 runnable 

"VM Periodic Task Thread" os_prio=0 tid=0x00007f101c239800 nid=0x2b0e waiting on condition 

JNI global references: 401

Heap
 PSYoungGen      total 143360K, used 41865K [0x00000000eab00000, 0x00000000f6c00000, 0x0000000100000000)
  eden space 131072K, 31% used [0x00000000eab00000,0x00000000ed3e2640,0x00000000f2b00000)
  from space 12288K, 0% used [0x00000000f3700000,0x00000000f3700000,0x00000000f4300000)
  to   space 12288K, 0% used [0x00000000f2b00000,0x00000000f2b00000,0x00000000f3700000)
 ParOldGen       total 106496K, used 19845K [0x00000000c0000000, 0x00000000c6800000, 0x00000000eab00000)
  object space 106496K, 18% used [0x00000000c0000000,0x00000000c1361410,0x00000000c6800000)
 Metaspace       used 38230K, capacity 38669K, committed 38872K, reserved 1083392K
  class space    used 4695K, capacity 4808K, committed 4864K, reserved 1048576K

**Comment by duncdrum** [comment]
I can not reproduce this, and haven't had problems with shutdown via dashboard in a long time. If this is still an issue feel free to reopen or create a new issue.

**Comment by dizzzz** [comment]
For me , the button always works
